models/targeting/nucleotide_insertion.py

```diff
@@ -0,0 +1,13 @@
+from random import randint
+
+# random insertion function
+# takes insertion size and produces random insertion string
+
+DNA_ALPHABET = "acgt"
```

Comment:
math3D constants.py is far away, I don't think we should import from such a distinct module

I'm ok having lone global constants for now, as we get more let's make a new constants.py or defaults.py in this module